### PR TITLE
media-plugins/imlib2_loaders: bump to 1.5.1 and remove -9999

### PR DESCRIPTION
**Pull request by juippis**
It installs .la files, but there is no pkgconfig used, and no static configure option. All the previous versions does the same. 
  https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Handling_Libtool_Archives 

Simple bump to latest version and update to EAPI7.

### CODE REVIEW

File: media-plugins/imlib2_loaders/imlib2_loaders-1.5.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	eet? ( dev-libs/efl[eet] )
+"
+
+src_configure() {
+	local myconf=(
+		$(use_enable eet)
+		$(use_enable xcf)
+	)
+
+	ECONF_SOURCE="${S}" \
```
Comment (mgorny) [inline media-plugins/imlib2_loaders/imlib2_loaders-1.5.1.ebuild]: This doesn't seem to do anything, does it?

Comment (juippis) [reply media-plugins/imlib2_loaders/imlib2_loaders-1.5.1.ebuild]: Probably not, took it from enlightenment.eclass. Tried without and it builds and installs fine. 

I took a closer look in configure phase and this package has static configure option after all, so Ill update the ebuild accordingly and push changes. It still doesnt seem to install .pc file, even though it checks for pkgconfig, so I guess some "if ! use static-libs" is needed?
>checking for x86_64-pc-linux-gnu-pkg-config... /usr/bin/x86_64-pc-linux-gnu-pkg-config
>checking pkg-config is at least version 0.9.0... yes

Comment (mgorny) [reply media-plugins/imlib2_loaders/imlib2_loaders-1.5.1.ebuild]: Well, unless I'm missing something this package installs plugins only, so static libraries have no value.

Comment (juippis) [reply media-plugins/imlib2_loaders/imlib2_loaders-1.5.1.ebuild]: Thats true. Ill ask before performing actions this time: I can safely add '--disable-static" and remove .la in every cases using
>find "${D}" -name '*.la' -delete || die

or just clean ebuild from everything related to static-libs and leave it to defaults?

Comment (mgorny) [reply media-plugins/imlib2_loaders/imlib2_loaders-1.5.1.ebuild]: Well, I'd have to use my favorite answer: it depends™ ;-). You need to pass `--disable-static` only if the build system enables static libraries by default. If it builds only shared libs, it is unnecesary. Removing `.la` files is desired most of the time. However, some very old plugin loaders actually use them. Remove them and check if everything still works.

**Comment by mgorny** [comment]
Thanks. I'll merge it.

**Pull request closed without merging**